content-v2/guides/userguides/exploring-projects.md

```diff
@@ -76,20 +76,18 @@ A partial example of a TSV file is listed below:
 
 ## Downloading Project DCP Generated Matrices
 
-Each project processed with DCP pipelines has DCP-generated matrices.
-
-To download Project matrices, navigate to the Project Information page and select the **"Project Matrices"** tab to the left. 
+Each project processed with DCP pipelines has DCP-generated matrices.To download Project matrices, navigate to the Project Information page and select the **"Project Matrices"** tab to the left. 
 
 Scroll to identify the relevant matrix and then select the download icon. 
 
 ![Project Matrices](../_images/dcp_generated_matrices.png "Project Matrices")
```

Comment:
Is this image going to change to one for DCP2 that has only the loom downloads? ie here https://data.humancellatlas.org/explore/projects/4a95101c-9ffc-4f30-a809-f04518a23803/expression-matrices

Reply:
@kbergin We did update it- it looks like it renders appropriately.

Reply:
Oh great, ok! It must have just rendered wrong on github. Thanks!